src/commoncode/date.py

```diff
@@ -35,16 +35,16 @@ def isoformat(utc_date):
     return datetime.isoformat(utc_date).replace('T', ' ')
 
 
-def get_file_mtime(location, iso=True):
+def get_file_mtime(location, ISO=True):
```

Comment:
We do not use uppercase for variable names.

Reply:
Will keep that in mind. Reverted the variable name changes.